### PR TITLE
feat: allow to disable actions defined in env variables

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
This will allow to define actions that has to be disable for tao instance

https://oat-sa.atlassian.net/browse/AUT-553

### CODE REVIEW

File: models/classes/featureFlag/ActionDisabledFeatureFlag.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+namespace oat\tao\model\featureFlag;
+
+use oat\oatbox\service\ConfigurableService;
+
+class ActionDisabledFeatureFlag extends ConfigurableService
+{
+    private const DISABLED_ACTIONS = 'DISABLED_ACTIONS';
+
+    public function isActionDisabled(string $feature): bool
```
Comment (mike-ionut-mihai-sandu-tao) [inline models/classes/featureFlag/ActionDisabledFeatureFlag.php:31]: as i remember last time we spoke about this with @gabrielfs7 everything is disabled unless we set it as active 
so creating a disabled list makes sense in a way that we want everything on unless we say so , 
so we don't have to create a list with all existing actions
let's have a call that will include Gabriel and Misha and white board this decision

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]

**Review by mike-ionut-mihai-sandu-tao — COMMENTED** [review]
i think we need some tests or at least a technical dept ticket

File: models/classes/featureFlag/ActionDisabledFeatureFlag.php (hunk 1 above)
Comment (gabrielfs7) [reply models/classes/featureFlag/ActionDisabledFeatureFlag.php:31]: So as I understood you want to disable actions from the Menu based on another env var.

Just a remind that they might still be available from mother parts. Example, disabling groups from menu do not hide them from delivery edit page. Is this ok? Might complicate even more the management...

Why no use the already existing https://github.com/oat-sa/tao-core/blob/master/models/classes/featureFlag/FeatureFlagChecker.php#L29
?

You could have feature flags called `FEATURE_FLAG_DISABLE_ACION_{ACTION_NAME}`

So you can reuse in another places.

But to be honest, I would like to understand why not manage this with ACL? If there is a new section not accessible by default user, then we would not map it on ACL. 

Wdyt?

Comment (mike-ionut-mihai-sandu-tao) [reply models/classes/featureFlag/ActionDisabledFeatureFlag.php:31]: we were able to split the ticket related to this feature and we will have a proper implementation in the next sprit that will cover a bit more cases

**Comment by mike-ionut-mihai-sandu-tao** [comment]
do we want to still keep this ? as i see we should go for a more general solution

**Pull request closed without merging**